erpnext/regional/report/fichier_des_ecritures_comptables_[fec]/fichier_des_ecritures_comptables_[fec].py

```diff
@@ -44,24 +43,116 @@ def set_account_currency(filters):
 
 def get_columns(filters):
 	columns = [
-		"JournalCode" + "::90",
-		"JournalLib" + "::90",
-		"EcritureNum" + ":Dynamic Link:90",
-		"EcritureDate" + "::90",
-		"CompteNum" + ":Link/Account:100",
-		"CompteLib" + ":Link/Account:200",
-		"CompAuxNum" + "::90",
-		"CompAuxLib" + "::90",
-		"PieceRef" + "::90",
-		"PieceDate" + "::90",
-		"EcritureLib" + "::90",
-		"Debit" + "::90",
-		"Credit" + "::90",
-		"EcritureLet" + "::90",
-		"DateLet" + "::90",
-		"ValidDate" + "::90",
-		"Montantdevise" + "::90",
-		"Idevise" + "::90",
+		{
+			"label": _("JournalCode"),
+			"fieldname": "JournalCode",
+			"fieldtype": "Data",
+			"width": 90,
+		},
+		{
+			"label": _("JournalLib"),
+			"fieldname": "JournalLib",
+			"fieldtype": "Data",
+			"width": 90,
+		},
+		{
+			"label": _("EcritureNum"),
+			"fieldname": "EcritureNum",
+			"fieldtype": "Dynamic Link",
```

Comment:
Dynamic Link usually needs `"options": "fieldname containing the doctype to link to"` 🤔

Reply:
... and your force push reverted my changes from the last review 🥲 usually you'd first pull existing changes, then add your new ones on top.

Reply:
@barredterra Sorry, I usually do the same, but I was sure that the reject push was because of develop merge from github and not local..., I didn't check you've update the PR directly

Reply:
I've change to data